WooCommerce/WooCommerceTests/Tools/IntegerInputFormatterTests.swift

```diff
@@ -3,7 +3,7 @@ import XCTest
 @testable import WooCommerce
 
 final class IntegerInputFormatterTests: XCTestCase {
-    private let formatter = IntegerInputFormatter()
+    private let formatter = IntegerInputFormatter(defaultValue: "0")
```

Comment:
since the negative prefix is an edge case, could we have some unit tests for the negative prefix (-) and also a negative number?

Reply:
Good suggestion! I added some test cases where the input it's just the minus sign `-` or multiple minus signs. There are other tests for negative numbers that I previously added in this PR https://github.com/woocommerce/woocommerce-ios/issues/2205. 👍